### PR TITLE
Add exploration effects for scouting and follow the expert

### PR DESCRIPTION
**Pull request by BernhardPosselt**

### CODE REVIEW

File: packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                    },
+                    {
+                        "label": "legendary",
+                        "sort": 3,
+                        "value": 4
+                    }
+                ],
+                "flag": "followTheExpertProficiency",
+                "key": "ChoiceSet",
+                "prompt": "Select expert's proficiency in skill"
```
Comment (Shandyan) [inline packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json:113]: The prompt should be in re-en.json

File: packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            {
+                "key": "FlatModifier",
+                "selector": "{item|flags.pf2e.rulesSelections.followTheExpertSkill}",
+                "slug": "followTheExpertCircumstanceModifier",
+                "type": "circumstance",
+                "value": "{item|flags.pf2e.rulesSelections.followTheExpertProficiency}"
+            }
+        ],
+        "source": {
+            "value": ""
```
Comment (Shandyan) [inline packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json:131]:
```suggestion
            "value": "Pathfinder Core Rulebook"
```

File: packs/data/exploration-effects.db/exploration-effect-scouting.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "selector": "initiative",
+                "type": "circumstance",
+                "value": 1
+            }
+        ],
+        "source": {
+            "value": ""
```
Comment (Shandyan) [inline packs/data/exploration-effects.db/exploration-effect-scouting.json:26]:
```suggestion
            "value": "Pathfinder Core Rulebook"
```

File: packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                    }
+                ],
+                "flag": "followTheExpertSkill",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Skill"
+            },
+            {
+                "choices": [
+                    {
+                        "label": "expert",
```
Comment (Shandyan) [inline packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json:96]:
```suggestion
                        "label": "PF2E.ProficiencyLevel2",
```

File: packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+                        "sort": 1,
+                        "value": 2
+                    },
+                    {
+                        "label": "master",
```
Comment (Shandyan) [inline packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json:101]:
```suggestion
                        "label": "PF2E.ProficiencyLevel3",
```

File: packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+                        "sort": 2,
+                        "value": 3
+                    },
+                    {
+                        "label": "legendary",
```
Comment (Shandyan) [inline packs/data/exploration-effects.db/exploration-effect-follow-the-expert.json:106]:
```suggestion
                        "label": "PF2E.ProficiencyLevel4",
```

File: packs/data/exploration-effects.db/exploration-effect-scouting-incredible-scout.json [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "selector": "initiative",
+                "type": "circumstance",
+                "value": 2
+            }
+        ],
+        "source": {
+            "value": ""
```
Comment (Shandyan) [inline packs/data/exploration-effects.db/exploration-effect-scouting-incredible-scout.json:26]:
```suggestion
            "value": "Pathfinder Advanced Player's Guide"
```

**Review by Shandyan — APPROVED** [review]

**Comment by BernhardPosselt** [comment]
For anyone merging: please squash merge to save me some headache later on :P

**Comment by stwlam** [comment]
> For anyone merging: please squash merge to save me some headache later on :P

Pull it all down and remake into single commit :)

**Comment by BernhardPosselt** [comment]
For you it's a drop down, for me it's rebase hell, so no ;D

**Comment by stwlam** [comment]
> For you it's a drop down, for me it's rebase hell, so no ;D

Do I look like some windows weenie to you? :stuck_out_tongue:

**Comment by stwlam** [comment]
Rename to "Effect: [Effect]"

**Pull request merged**